src/tests/members.test.js

```diff
@@ -1,5 +1,132 @@
-// Basic test to avoid conflicts
-// REMOVE THIS TEST AND MAKE YOURS!
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import Member from '../models/Member';
+import memberSeed from '../seeds/members';
+
+beforeEach(async () => {
+  await Member.collection.insertMany(memberSeed);
+});
+
+afterEach(async () => {
+  await Member.collection.deleteMany();
+});
+
+// eslint-disable-next-line
+const mockMemberSeed = memberSeed[0]._id;
+const mockMemberGood = {
+  firstName: 'Juan',
+  lastName: 'Perez',
+  dni: 41317666,
+  email: 'testeando@Radium.com',
+  phone: 1763060997,
+  password: 'd9vUeOyuKs',
+};
+const mockMemberEmpty = {
+};
+const mockMemberBadValidations = {
+  firstName: 23,
+  lastName: 'Perezaaaaaaaaaaaaaaaaaasssssssssssssssssssassadsasdaaaaaa',
+  dni: '41317666',
+  email: 'testeandoRadium.com',
+  phone: 176306099700000000,
+  password: 'd9vUeOyuKsasdasd@',
+};
+const mockIdBad = {
+  id: '64691519333281ea60b8069a',
+};
```

Comment:
Please change this objecto to 
```suggestion
const mockIdBad = '64691519333281ea60b8069a';
```